package.json

```
@@ -2,7 +2,7 @@
   "author": "Larry Myers",
   "name": "jasmine-reporters",
   "description": "Reporters for the Jasmine BDD Framework",
-  "version": "2.3.2",
+  "version": "2.3.3",
```

This is the patch version, I think @larrymyers  wants the minor version to be bumped instead?

```suggestion
  "version": "2.4.0",
```